curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -306,15 +329,17 @@ int Warmup(const std::string& name, const std::string& value) {
     }
     int ret = 0;
     switch (curvefs::client::common::GetWarmupOpType(opTypePath[0])) {
-        case curvefs::client::common::WarmupOpType::kWarmupOpAdd:
-            ret = AddWarmupTask(opTypePath[1], opTypePath[2]);
-            if (ret != 0) {
-                LOG(ERROR) << name << " has invalid xattr value " << value;
-            }
-            break;
-        default:
+    case curvefs::client::common::WarmupOpType::kWarmupOpAdd:
+        ret =
+            AddWarmupTask(curvefs::client::common::GetWarmupType(opTypePath[1]),
+                          key, opTypePath[2]);
+        if (ret != 0) {
             LOG(ERROR) << name << " has invalid xattr value " << value;
-            ret = ERANGE;
+        }
+        break;
+    default:
+        LOG(ERROR) << name << " has invalid xattr value " << value;
+        ret = ERANGE;
```

Comment:
> Should we replace all previous `ERANGE` with `EOPNOTSUPP`?

Here, set the extended attribute curvefs.warmup.op to add xxx, and then parse whether it is the content of add. I think the wrong parsing should be due to setting errors rather than unsupported.